src/main/java/emu/grasscutter/command/commands/ClearCommand.java

```diff
@@ -42,7 +42,8 @@ public void execute(Player sender, Player targetPlayer, List<String> args) {
             case "art" -> {
             	toDelete = playerInventory.getItems().values().stream()
                         .filter(item -> item.getItemType() == ItemType.ITEM_RELIQUARY)
-                        .filter(item -> item.getLevel() == 1 && item.getExp() == 0)
```

Comment:
Note this changes logic of `clearall`

Reply:
Isn't this the right change? Only sacred relics worn and locked will not be deleted. However, the Readme. md content in English has not been corrected.